runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscoreConfig.java

```diff
@@ -78,4 +78,12 @@ default boolean autocomplete()
 	{
 		return true;
 	}
-}
\ No newline at end of file
+
+	@ConfigItem(
+		position = 5,
+		keyName = "lookupOnInteraction",
+		name = "Lookup players on interaction",
+		description = "Display a combat stat comparison panel on player interaction. (follow, trade, challenge, attack, etc.)"
+	)
+	default boolean lookupOnInteraction() { return false; }
```

Comment:
Travis wants the left curly on line 88 col 40 to be on a new line :/
```c
[ERROR] src/main/java/net/runelite/client/plugins/hiscore/HiscoreConfig.java:[88,47] (blocks) LeftCurly: '{' at column 40 should be on a new line.
```

```java
default boolean lookupOnInteraction()
{
    return false;
}
```

Aside from that checkstyle, this looks really useful!

Reply:
D'oh, I copied the formatting from the collapsed instances above.

Reply:
haha :p